.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+name: Package and Publish to Visual Studio Marketplace
+
+on:
+  push:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Set Git configuration
+        run: |
+            git config user.name "${GITHUB_ACTOR}"
+            git config user.email "${GITHUB_ACTOR}@bots.github.com"
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Bump version and create Git tag
+        uses: Aqovia/npm-version-increment@v1.0.4
+        id: version
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        
+      - name: Extract extensionPack from package.json
+        run: |
+            # Get the "extensionPack" section from package.json and save it in a file
+            node -p "JSON.stringify(require('./package.json').extensionPack, null, 2)" > extensionPack.json
+            # Output the contents of extensionPack.json to the build log
+            #cat extensionPack.json
+  
+      - name: Update README.md
+        shell: pwsh
+        run: |
+            # Read the extensions from extensionPack.json and format them as bullet points
+            $extensions = Get-Content -Raw -Path "extensionPack.json" | ConvertFrom-Json
+            $formatted_extensions = $extensions -join "`n- "
```

Comment:
Can you find the name and URL of the extension here?
For example instead of  `hnw.vscode-auto-open-markdown-preview`, we have:
* [Auto-Open Markdown Preview](https://marketplace.visualstudio.com/items?itemName=hnw.vscode-auto-open-markdown-preview)

I guess the URL is easy,: https://marketplace.visualstudio.com/items?itemName=<Extension ID>

Reply:
You can also find the display name:

```
vsce show hnw.vscode-auto-open-markdown-preview --json | ConvertFrom-Json | Select -ExpandProperty displayName
```

Reply:
Ok, so you want it to display in the readme, the friendly name and the URL of the extension?